libs/logic-apps-shared/src/designer-client-services/lib/base/search.ts

```diff
@@ -248,6 +254,14 @@ export abstract class BaseSearchService implements ISearchService {
     return connectors;
   }
 
+  public removeUnsupportedOperations(operations: DiscoveryOpArray): DiscoveryOpArray {
+    return operations.filter((operation) => !this._unsupportedConnectorIds.includes(operation?.properties?.api?.id));
```

Comment:
I would suggest to normalize the connectorIds in your list and from the api.id to do the includes.. because it is a resource I'd and even if they have different casing they would be same resource so you need to be case insensitive here.

Reply:
Can you help clarify 'normalizing the connectorIds in your list and from the api.id' - what would you mean by normalize here, to Lowercase those?